conf/config-default.yaml

```diff
@@ -340,6 +340,7 @@ plugins:                          # plugin list (sorted by priority)
   - response-rewrite               # priority: 899
   #- dubbo-proxy                   # priority: 507
   - grpc-transcode                 # priority: 506
+  - azure-functions                # priority: 505
```

Comment:
Better to put it ahead of serverless-post-function

Reply:
What should be the priority value, negative?

Reply:
priority: -1900?